src/components/dateCourse/timeline.tsx

```
@@ -10,45 +10,52 @@ import Cash from '@/assets/icons/cash_Blank.svg?react';
-                        <KeywordButton tag={menu!} />
+                    <div className="flex flex-col gap-[8px] lg:w-[50%] max-h-fit justify-around items-start">
+                        {signatureDish && (
+                            <div className="flex w-full flex-col ">
+                                <div className="flex gap-[8px] font-body1 select-none text-center items-center h-[24px] w-full">
+                                    <div className="flex text-center h-full">WithTime Pick</div>
+                                    <CheckSuccess stroke={'#000000'} />
+                                </div>
+                                <KeywordButton tag={signatureDish.name!} />
+                            </div>
+                        )}
+                        {image && <img src={image} className="w-[80%] self-start" />}
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)